doc/THIRD-PARTY-LICENSES

```diff
@@ -1,16 +1,413 @@
-_: BSD-3-Clause
-cmake/Modules/DownloadProject.cmake: mit
-cmake/Modules/DownloadProject.CMakeLists.cmake.in: mit
-cmake/Modules/DownloadProjectLicense.txt: mit
-cmake/Modules/FindGLib.cmake: mit
-scripts/realpath: mit
-src/plugins/ni/nickel-1.1.0/_: zlib
-src/plugins/ni/nickel-1.1.0/src/hash.c: public-domain
-src/tools/website/public/assets/fonts/font-awesome/_: ofl-1.1
-src/tools/website/public/assets/fonts/lato/_: ofl-1.1
-src/tools/website/public/assets/fonts/libre-franklin/\*: ofl-1.1
-src/tools/website/resources/assets/skin/base/fonts/font-awesome/font-awesome.less: mit, ofl-1.1
-src/tools/website/resources/assets/skin/base/util/space.less: mit
-src/tools/web/client/public/Roboto-Light.ttf: apache-2.0
-src/tools/web/client/public/Roboto-Regular.ttf: apache-2.0
-src/tools/kdb/gen/mustache.hpp: BSL-1.0
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Elektra
+Upstream-Contact: Markus Raab <elektra@markus-raab.org>
+Source: http://www.libelektra.org/ftp/elektra/releases/
+
+Files: *
+Copyright: 2003-2004, 2006-2008, 2010, 2013-2014 Markus Raab <elektra@markus-raab.org>
```

Comment:
Is it possible to simply refer to doc/AUTHORS.md here? (Btw. doc/AUTHORS can be removed, I do not think it is used anymore).

And please feel free to add yourself also to doc/AUTHORS.md.

Reply:
I couldn't find an example where a specific file was referenced, but we could simply write "libelektra contributors". [Neovim](https://metadata.ftp-master.debian.org/changelogs//main/n/neovim/neovim_0.3.4-3_copyright) for example is doing it this way.

Reply:
Yes, good idea. I would say we can even write
`Elektra contributors (https://www.libelektra.org/developers/authors)`

It is free text, so the weirdest things are there. Also links are quite common there. (I used `grep "Copyright:.*/" /usr/share/doc/*/copyright` to find out.)